VUE2/src/build.xml

```diff
@@ -63,7 +63,7 @@
   <property name="test.home"   value="${top.dir}/test" />
   <property name="test.lib"   value="${test.home}/lib" />
   <property name="junit.jar"   value="${test.lib}/junit.jar" />
-  <property name="target.version" value="1.6"/>
+  <property name="target.version" value="1.7"/>
```

Comment:
It looks like you are targeting "java 7"... as opposed to "java 12".
My standard JVM these days is the LTS java 11... which results in a build failure on this pull-request.